rewards.staging.mainnet.env

```diff
@@ -0,0 +1,7 @@
+# config
+export RPC_URL=https://eth.llamarpc.com
+export CHAIN=mainnet
+export FACTORY=0x290CAB97a312164Ccf095d75D6175dF1C4A0a25F # see symbiotic docs
```

Comment:
Current documentation is not updated yet with a rewards factory of version 2, here it is - [0xFEB871581C2ab2e1EEe6f7dDC7e6246cFa087A23](https://etherscan.io/address/0xFEB871581C2ab2e1EEe6f7dDC7e6246cFa087A23)